### ISSUE TITLE
WebInstaller should as for elevated rights

### ISSUE THREAD

**Issue by mWaltari** [Memnarch/Delphinus#37]
At least in my environment it does not work if I do not run installer as Administrator.

Would make things easier if the installer would ask at startup that it needs to be elevated.

**Comment by Memnarch** [comment]
I know, but i did not add a manifest on purpose. Since i haven't bought a proper certificate yet. I am not a fan of unsigned administrative manifests.

However, it should work, if you install to a location the current user has write access to. The default setting of ProgramFiles(x86) is the culprit here.

**Comment by mWaltari** [comment]
Oh yes... That is an real problem, the certificate. 

Can we detect that program is not running on Elevated or similar rights. And program could tell user, that try running me as Admin. That would be more than enough for me.

As a kludge testing read/write etc access to installation folder would do. But not very elegant, there must be better way.

**Comment by Memnarch** [comment]
Yes, good idea. it should at least check for read/write access for the given folder.
